.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,45 @@
+name: Building Docker images
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
This means you're pipeline is going to build and push `latest` for every PR created and push to master.

Reply:
I think this is still unresolved. We shouldn't push any image to dockerhub on PR, as it would mean anyone can push images freely to dockerhub under our account (by modifying `Dockerfile` in the PR).  
The docker image should only be pushed on PR merges to master.